src/eve-server/system/Container.cpp

```diff
@@ -382,16 +370,15 @@ PyDict *ContainerSE::MakeSlimItem() {
         slim->SetItemString("corpID",           IsCorp(m_corpID) ? new PyInt(m_corpID) : PyStatic.NewNone());
         slim->SetItemString("allianceID",       IsAlliance(m_allyID) ? new PyInt(m_allyID) : PyStatic.NewNone());
         slim->SetItemString("warFactionID",     IsFaction(m_warID) ? new PyInt(m_warID) : PyStatic.NewNone());
-        if (m_contRef->IsAnchored())        // not sure if this is right...testing
-            slim->SetItemString("structureState",       new PyInt(EVEPOS::StructureState::Anchored));
+        slim->SetItemString("structureState",       new PyInt((m_data.state == EVEPOS::StructureState::Anchored)?1:0));
 
     if (is_log_enabled(DESTINY__DEBUG)) {
         _log( DESTINY__DEBUG, "ContainerSE::MakeSlimItem() - %s(%u)", GetName(), GetID());
         slim->Dump(DESTINY__DEBUG, "     ");
     }
 
     return slim;
-}
+} */
```

Comment:
I'd say completely remove all of this (assuming that Structure handles all of the same strings correctly). There's no reason to keep it around.

Reply:
Agreed, was waiting for a go about it, will happend next commit. Now I have to figure out why can't I unanchore ... I still strugle with effects ... before preventing tractor beams to tract anchored containers.